feedthefox/dashboard/templates/jinja2/dashboard.html

```diff
@@ -5,7 +5,10 @@
 {% endblock %}
 
 {% block content %}
-  <h3>Dashboard</h3>
+  <h3 class="dashboard-title">Dashboard</h3>
+  <i class="fa fa-info-circle"
+     title="Help make Firefox OS better. Contribute to prioritizing features or defects by voting for those you most want to see worked on. Report a feature or defect by clicking the ‘+’ button on the right.">
```

Comment:
It works here. I get this error when I actually press the '+' button (although it still works). But this is from an old merge, so I investigate further.